.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
```

Comment:
I just want to auto-update for github-action, so it will let the dependabot create a PR to work for like that:

```diff
-       uses: actions/checkout@v3
+       uses: actions/checkout@v4
```

If you also want cargo to automatically upgrade dependencies, I'd be happy to add the configuration.

Reply:
I think the focus of xline at this stage is on the development of functions, so automatically updating dependencies may not be that important. 
No strong opinion from me.

Reply:
IMO, automatically upgrade cargo dependencies is a good practice.

Reply:
@Phoenix500526 
 And you also want to schedule with `daily` for it?

Reply:
cc @bsbds  ^^^

Reply:
My suggestion is to use `monthly` to reduce noise as much as possible at this stage if you want to enable aoto-update for cargo dependenices, and you can update it to `weekly` later.

Reply:
Okay, you've convinced me. Daily might be too frequent; weekly is a better option.